include/rocksdb/table.h

```diff
@@ -108,6 +108,9 @@ struct BlockBasedTableOptions {
   };
 
   DataBlockIndexType data_block_index_type = kDataBlockBinarySearch;
+  // #entries/#buckets
+  double data_block_hash_table_util_ratio = 0.75;
```

Comment:
I rethink about this option. `DataBlockIndexType data_block_index_type` is a little bit confusing. We always have binary index and hash index is just an add on. So setting `data_block_index_type = kDataBlockHashSearch` may be confusing. I know the pattern is copied from `index_type` but it still doesn't feel like very well to me. Maybe call it `kDataBlockBinaryAndHash` is less confusing.

Also the `DataBlock` part of `kDataBlockBinarySearch` and `kDataBlockHashSearch` feels unnecessary to me. Maybe we should remove them.

Reply:
fixed `kDataBlockHashSearch` to `kDataBlockBinaryAndHash`.
BTW, if remove the `DataBlock` part of `kDataBlockBinarySearch`, it has a conflicting name with the `IndexType`:
https://github.com/facebook/rocksdb/blob/5.15.fb/include/rocksdb/table.h#L87-L101